src/main/webapp/WEB-INF/jsp/help.jsp

```diff
@@ -0,0 +1,181 @@
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <c:set var="context" value="${pageContext.request.contextPath}"/>
+
+    <link rel="stylesheet" href="${context}/bootstrap/css/bootstrap.min.css"/>
+    <link rel="stylesheet" href="${context}/bootstrap/css/bootstrap-theme.css"/>
+    <link rel="stylesheet" href="${context}/bootstrap/css/bootstrap-datepicker.min.css"/>
+    <link rel="stylesheet" href="${context}/stylesheets/jquery-ui.min.css"/>
+    <link rel="stylesheet" href="${context}/stylesheets/style.css"/>
+    <link rel="stylesheet" href="${context}/css/buttonOverrides.css"/>
+
+    <script src="${context}/js/jquery.2.1.3.min.js"></script>
+    <script src="${context}/js/jquery-ui.min.js"></script>
+    <script src="${context}/bootstrap/js/bootstrap.min.js"></script>
+    <script src="${context}/js/jquery.confirm.js"></script>
+    <script src="${context}/js/scripts.js"></script>
+    <title>Help</title>
+</head>
+<body>
+<nav class="navbar navbar-default">
+    <div class="container-fluid">
+        <div class="navbar-header">
+            <a class="navbar-brand" href="${context}/roster/${selectedSectionId}">K-State Attendance</a>
+        </div>
+        <ul class="nav navbar-nav">
+            <li><a id="classSetupLink" href="${context}/courseConfiguration/${selectedSectionId}">Setup</a></li>
+            <li><a id="attendanceSummaryLink" href="${context}/attendanceSummary/${selectedSectionId}">Attendance
+                Summary</a></li>
+            <li><a id="rosterLink" href="${context}/roster/${selectedSectionId}">Class Roster</a></li>
+            <li class="active"><a id="helpLink" href="${context}/help/${selectedSectionId}">Help</a></li>
+        </ul>
+    </div>
+</nav>
+
+<h1>Start Up Guide for K-State Attendance</h1>
+<hr>
+
+<div class="panel-group form-div" id="accordion" role="tablist" aria-multiselectable="true">
+    <div class="panel panel-default">
+        <div class="panel-heading" role="tab" id="headingOne">
+            <h4 class="panel-title">
+                <a class="collapsed" role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseOne"
+                   aria-expanded="false" aria-controls="collapseOne">
+                Saving a Setup
+                </a>
+            </h4>
+        </div>
+        <div id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
+            <div class="panel-body">
+                <img src="${context}/img/Setup.png" border="5">
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this deprecated "border" attribute. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AS1827)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add an "alt" attribute to this image. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3AImgWithoutAltCheck)